draft-ietf-idr-deprecate-as-set-confed-set.xml

```diff
@@ -504,9 +501,19 @@ R4 - 192.0.2.192/26 AS_PATH "64504 64501"
                       overlap at the origin AS.">
         <t>Receive R1.  Aggregate 192.0.2.0/24 AS_PATH "64501"</t>
 
-        <t>Receive R4.  Aggregate 192.0.2.0/24 AS_PATH "64504 [ 64501 ]"</t>
+        <t>Receive R4.  Aggregate 192.0.2.0/24 AS_PATH "[ 64504 64501 ]"</t>
+
+<!-- 
+XXX KS: I make a correction above from "64504 [ 64501 ]" 
+to "[ 64504 64501 ] because "[ 64504 ] 64501" does not follow
+from algorithm in RFC 4271. 64504 is not common to both
+contributing paths. So it must also be included in the AS_SET.
+-->    
+
+<!-- XXX KS: This is replaced (as below):  <t>If brief aggregation is in use, 
+the AS_PATH is truncated to "64504".</t>  -->

```

Comment:
The error here you're noting is valid.
The intent is to have an example with a common leading AS with trailing AS that is thrown into a set that is discarded.  You can either update this in your diff, or leave it for me to update in mine once we merge your stuff in.

Reply:
Yes, your origin example B.2 didn't follow this condition from rfc 4271:
           - all tuples of type AS_SEQUENCE in the aggregated AS_PATH
             SHALL appear in all of the AS_PATHs in the initial set of
             routes to be aggregated. 
As I see it B.3 is an "an example with a common leading AS with trailing AS that is thrown into a set that is discarded."
I leave this up to you to sort out.